obal/data/modules/srpm.py

```diff
@@ -0,0 +1,122 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import subprocess
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.chdir import chdir # pylint:disable=import-error,no-name-in-module
+
+def run_command(command):
+    """
+    Run a system command
+    """
+    return subprocess.check_output(
+        ' '.join(command),
```

Comment:
the `join` is unnecessary, `subprocess` can handle lists just fine.

Reply:
My experience so far has been that it does not handle them well when combined with calling rpmbuild and specifyin macros that require escaping. I would love it if someone figures out the magic. I spent more time than I should have going round and round with Python subprocess trying to call rpmbuild where macros where defined.